### DISCUSSION TITLE
Log Request only

### DISCUSSION THREAD

**Discussion by loranger** [bilfeldt/laravel-http-client-logger#23] [Q&A] [answered]
Hi,

Is there a simple way to log request only ?

**Comment by bilfeldt** [comment, marked as answer]
> Hi,
> 
> Is there a simple way to log request only ?

@loranger if you just want a few request to be logged like that, then you should use the [conditional logger](https://github.com/bilfeldt/laravel-http-client-logger#specifying-a-logger) where you can specify your own class.

if on the other hand you want **all** requests to log like that, then you can implement your own class and register that in the config `logger`. You could consider making a new class that extends [`Bilfeldt\LaravelHttpClientLogger\HttpLogger`](`https://github.com/bilfeldt/laravel-http-client-logger/blob/master/src/HttpLogger.php`) where it overrides the `log` method

**Reply by loranger** [reply]
I was looking for the second solution, thanks !
